### PR TITLE
Fix memory leak on swaybar

### PR DESCRIPTION
**Pull request by bpinto**
Code from @NilsBrause

Ref #1820

### CODE REVIEW

File: swaybar/i3bar.c [hunk 1]
```diff
@@ -104,6 +104,7 @@ static bool i3bar_parse_json(struct status_line *status, const char *text) {
 			json_object_get_int(border_right) : 1;
 		wl_list_insert(&status->blocks, &block->link);
 	}
+  json_object_put(results);
```
Comment (ascent12) [inline swaybar/i3bar.c]: Wrong indentation.

Comment (bpinto) [reply swaybar/i3bar.c]: In a single line commit I was able to get an issue... that's a new mark in my life :joy: 

Fixed it, I have my vim configured to use spaces and not tabs. Got it to use a tab now.

**Comment by bpinto** [comment]
@martinetd :100: 

I am not a C programmer and I don't even know (yet) how to get information about leaks - I might try to work on these other leaks you referenced. It was a surprise to me that a method named `json_object_put` would do anything to free memory. :joy: 

I just noticed that code was there on a comment and felt I could do something other than ignore it by packaging that single-line patch into a PR. And here it is, after an indentation issue. :grimacing:

**Comment by emersion** [comment]
3 approvals for a one-line fix, I can't leave this unmerged.

Thanks!

**Pull request merged**